keras_cv/losses/iou_loss.py

```diff
@@ -23,11 +23,13 @@ class IoULoss(keras.losses.Loss):
     """Implements the IoU Loss
 
     IoU loss is commonly used for object detection. This loss aims to directly
```

Comment:
How about instead we just say "IoU (Intersection over Union) loss ..." and scrap the other changes?